core/src/main/resources/jenkins/model/Jenkins/projectRelationship.jelly

```diff
@@ -26,73 +26,67 @@ THE SOFTWARE.
   Displays two projects side by side and show their relationship
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:layout title="${%Project Relationship}">
-    <st:include page="sidepanel.jelly" />
-    <l:breadcrumb title="${%Project Relationship}" />
+    <st:include page="sidepanel.jelly"/>
+    <l:breadcrumb title="${%Project Relationship}"/>
 
     <l:main-panel>
       <l:app-bar title="${%Project Relationship}">
-        <t:help href="projectRelationship-help" />
+        <t:help href="projectRelationship-help"/>
       </l:app-bar>
 
+      <p>
+        ${%body}
+      </p>
+
       <form action="projectRelationship" method="get">
-        <table width="100%">
-          <tr>
-            <j:set var="names" value="${app.jobNames}" />
-            <td style="text-align:right;">
-              ${%upstream project}:
-              <f:editableComboBox id="lhs" name="lhs" value="${request.getParameter('lhs')}" items="${names}" />
-            </td>
-            <td style="width:32px; text-align:center;"><l:icon class="icon-next icon-md"/></td>
-            <td>
-              ${%downstream project}:
-              <f:editableComboBox id="rhs" name="rhs" value="${request.getParameter('rhs')}" items="${names}" />
-            </td>
-          </tr>
-          <tr>
-            <td colspan="3" style="text-align:right">
-              <f:submit value="${%Compare}" />
-            </td>
-          </tr>
+        <div class="jenkins-form-item">
+          <label class="jenkins-form-label" for="lhs">${%upstream project}:</label>
+          <f:comboBox id="lhs" fillUrl="fillJobNameItems" name="lhs" value="${request.getParameter('lhs')}"/>
```

Comment:
```suggestion
          <f:combobox id="lhs" fillUrl="fillJobNameItems" name="lhs" value="${request.getParameter('lhs')}"/>
```

core-pr-tester fails to render these (they're in the page source verbatim), my guess is file system case sensitivity.